waveparser_test.go

```diff
@@ -100,3 +103,52 @@ func TestParseWAV(t *testing.T) {
 		}
 	}
 }
+
+func TestSignedInt16LittleEndianSamples(t *testing.T) {
+
+	wav, err := Load("testdata/audios/sint16le.wav")
+	assertNoError(t, err)
+
+	samples, err := wav.Int16LESamples()
+	assertNoError(t, err)
+
+	gotbuf := &bytes.Buffer{}
+	err = binary.Write(gotbuf, binary.LittleEndian, samples)
+	assertNoError(t, err)
+
+	expected, err := ioutil.ReadFile("testdata/audios/sint16le.raw")
```

Comment:
if you say it's right then I believe =)

Reply:
uaheuhuehuah

you can open the audio on audacity with "import raw audio"..no need for trust =P